cheddar/cheddar-integration-aws/src/main/java/com/clicktravel/infrastructure/persistence/aws/cloudsearch/AmazonCloudSearchDomainClientBuilder.java

```diff
@@ -23,7 +23,8 @@
 public class AmazonCloudSearchDomainClientBuilder {
 
     public static AmazonCloudSearchDomain build(final AWSCredentials awsCredentials, final String endpoint) {
```

Comment:
That approach would simply add complexity to the calling method, `CloudSearchEngine#cacheDomainEndpoints()` for no gain. This code will need to be revisited anyway to remove all mention of `AwsCredentials` once existing applications are migrated to not build CloudSearch domains on startup.